src/Extensions/CommentNotifier.php

```diff
@@ -48,12 +48,13 @@ public function notifyCommentRecipient($comment, $parent, $recipient)
         $subject = $parent->notificationSubject($comment, $recipient);
         $sender = $parent->notificationSender($comment, $recipient);
         $template = $parent->notificationTemplate($comment, $recipient);
+        $to = ($recipient instanceof Member) ? $recipient->Email : $recipient;
```

Comment:
Could you also update the docblock @param above to `* @param Member|string|array $recipient`

Reply:
Done